src/csv2dataset.py

```diff
@@ -0,0 +1,26 @@
+import csv
+import glob
+import pandas as pd
+
+# change scv to list
+def csv2list(path):
```

Comment:
If possible, it's better not to convert it to list and just leave it as databases